docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+services:
+  apache:
+    build:
+      context: .
+      dockerfile: .docker/Dockerfile
+    ports:
+      - "8080:80"
+    volumes:
+      - .:/app
+  db:
+    image: mariadb:10.5.1
```

Comment:
Why not use MySQL?

Reply:
force of habit really - I'm open to changing the versions of php and db as appropriate also.

Reply:
I think it should use the same database as the one used in production and mentioned in the setup instructions: https://github.com/CosmoQuestX/CSB7.0#11-setup-lamp-server

Reply:
Current CosmoQuest uses MariaDB afaict. Instructions might not reflect this since they have some history. But maybe @starstryder can clear that up.

Reply:
Production is MariaDB, but currently you can go back and forth without pain.